lib/src/database/db.dart

```diff
@@ -209,7 +210,7 @@ class Db {
       }
 
       if (connection == null) {
-        connection = _masterConnection;
+        connection = _masterConnectionVerified;
```

Comment:
@Pacane I can make test, if you help to mock slave connection.
What do I need at the moment for successful merge to master?